mix.exs

```diff
@@ -74,8 +76,9 @@ defmodule Cadet.Mixfile do
       {:distillery, "~> 1.5.3", runtime: false},
       {:excoveralls, "~> 0.8", only: :test},
       {:exvcr, "~> 0.10", only: :test},
+      {:faker, "~> 0.10", only: [:test, :dev]},
```

Comment:
Lol this triggers me. Can change to `[:dev, :test]`

Reply:
done